puppet/zulip_ops/manifests/profile/teleport.pp

```diff
@@ -0,0 +1,25 @@
+class zulip_ops::profile::teleport {
+  include zulip_ops::profile::base
+  include zulip_ops::teleport::base
+
+  file { '/etc/teleport_server.yaml':
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0644',
+    source => 'puppet:///modules/zulip_ops/teleport_server.yaml',
+  }
+  file { "${zulip::common::supervisor_conf_dir}/teleport_server.conf":
+    ensure  => file,
+    require => [ Package[supervisor], Package[teleport], File['/etc/teleport_server.yaml'] ],
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0644',
+    source  => 'puppet:///modules/zulip_ops/supervisor/conf.d/teleport_server.conf',
+    notify  => Service[$zulip::common::supervisor_service],
+  }
+
+  zulip_ops::firewall_allow { 'teleport_server_ui': port => 443 }
+  zulip_ops::firewall_allow { 'teleport_server_proxy': port => 3023 }
+  zulip_ops::firewall_allow { 'teleport_server_reverse': port => 3024 }
```

Comment:
I don't see where port `3024` appears in the configuration file above; might be worth adding it for some self-documentation.